phenotrex/cli/plot.py

```diff
@@ -66,26 +72,87 @@ def shap_summary(out, n_max_features, title, **kwargs):
               help='The prefix to generated SHAP force plots.')
-@click.option('--n_samples', type=int, default=None,
+@click.option('--out_summary', default=None, type=click.Path(dir_okay=False),
+              help='The path to save the generated summary at.')
+@click.option('--n_max_features', type=int, default=20,
+              help='The number of top most important features (by absolute SHAP value) to plot.')
+@click.option('--n_samples', type=str, default='auto',
               help='The nsamples parameter of SHAP. '
                    'Only used by models which utilize a `shap.KernelExplainer` (e.g. TrexSVM).')
 @click.option('--verb', is_flag=True)
-def shap_force(out_prefix, **kwargs):
+def shap_force(out_prefix, out_summary, n_max_features, **kwargs):
     """
     Generate SHAP force plots for each sample (passed either as FASTA files or as genotype file).
     All plots will be saved at the path `{out_prefix}_{sample_identifier}_force_plot.png`.
-    All non-existent directories in the output prefix path will be created as needed.
+    All non-existent directories in the out_prefix path will be created as needed.
+    """
+    import matplotlib as mpl
+    mpl.use('Agg')
+
+    from .generic_func import generic_compute_shaps
+    from phenotrex.util.plotting import shap_force_plots
+
+    sh, gr = generic_compute_shaps(**kwargs)
+    shap_force_plots(
+        gr=gr,
+        sh=sh,
+        out_prefix=out_prefix,
+        n_max_features=n_max_features,
+        out_individual_summary=out_summary
+    )
+
+
+@plot.command('shap-full', short_help='Create all SHAP plots and summaries.')
+@click.argument('fasta_files', type=click.Path(exists=True), nargs=-1)
+@click.option('--genotype', type=click.Path(exists=True),
+              required=False, help='Input genotype file.')
+@click.option('--classifier', required=True, type=click.Path(exists=True),
+              help='Path of pickled classifier file.')
+@click.option('--force_plot_prefix', type=str, default='force_plots/',
+              help='The prefix to generated per-sample SHAP force plots.')
+@click.option('--out_force_file', default='per_sample_forces.tsv', type=click.Path(dir_okay=False),
+              help='The path to save the generated per-sample SHAP force file.')
+@click.option('--out_summary_plot', default='shap_summary.png', type=click.Path(dir_okay=False),
+              help='The path to save the generated summary plot at.')
+@click.option('--out_summary_file', default='shap_summary.tsv', type=click.Path(dir_okay=False),
+              help='The path to save the generated summary file at.')
+@click.option('--summary_plot_title', type=str, default='SHAP Summary')
+@click.option('--n_max_features', type=int, default=30,
+              help='The number of top most important features (by absolute SHAP value) to plot.')
+@click.option('--n_samples', type=str, default='auto',
+              help='The nsamples parameter of SHAP. '
```

Comment:
Was that the variable we recently renamed to `n_samples`? In this case:
```suggestion
              help='The n_samples parameter of SHAP. '
```
EDIT: I see, SHAP itself uses nsamples. Users probably don't know that, though.